### PR TITLE
[HACK-112] Clarify Pentest Methodologies

### PR DESCRIPTION
**Pull request by mjang-cobalt**
Grahame, I want to deprecate our Zendesk articles related to Pentest Methodologies, ref https://cobaltio.zendesk.com/hc/en-us/sections/360012503971-Cobalt-Pentest-Methodologies

But before I deprecate the Zendesk content, I wanted to make sure that we had all needed content in our Product Docs (here).

I've also X checked against our internal Confluence docs on the subject.

It's OK with me if you want to wait until next week to review this.

### CODE REVIEW

**Comment by ana-dashuk-cobalt** [comment]
Looks good for me.

File: content/en/Getting started/Pentest objectives/Methodologies/api-methodologies.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ and Testing Guide. Our team takes the following steps to ensure full coverage:
 
 - Target scope reconnaissance
 - Business and application logic mapping
-- Automated web crawling and web scanner configuration tweaking
+- Authenticated web crawling and web scanner configuration tweaking
```
[the remark this answers is not in the document]
Comment (mjang) [reply content/en/Getting started/Pentest objectives/Methodologies/api-methodologies.md:40]: Now that I read the difference between this and our [internal doc](https://zombie.atlassian.net/wiki/spaces/PEN/pages/2137622499/Web+Application+Pentesting)

I realize I should have changed the next line instead. 

Good catch, thank you

```suggestion
- Automated web crawling and web scanner configuration tweaking
```

File: content/en/Getting started/Pentest objectives/Methodologies/api-methodologies.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ and Testing Guide. Our team takes the following steps to ensure full coverage:
...
 - Automated vulnerability scanning
```
Comment (mjang) [inline content/en/Getting started/Pentest objectives/Methodologies/api-methodologies.md:41]:
```suggestion
- Authenticated vulnerability scanning
```

File: content/en/Getting started/Pentest objectives/Methodologies/web-methodologies.md [hunk 3]
```diff
@@ -44,7 +44,7 @@ full coverage:
 
 - Target scope reconnaissance
 - Business and application logic mapping
-- Automated web crawling and web scanner configuration tweaking
+- Authenticated web crawling and web scanner configuration tweaking
 - Automated vulnerability scanning
```
Comment (mjang-cobalt) [inline content/en/Getting started/Pentest objectives/Methodologies/web-methodologies.md:48]:
```suggestion
- Authenticated vulnerability scanning
```

File: content/en/Getting started/Pentest objectives/Methodologies/web-methodologies.md [hunk 4]
```diff
@@ -44,7 +44,7 @@ full coverage:
 
 - Target scope reconnaissance
 - Business and application logic mapping
-- Automated web crawling and web scanner configuration tweaking
+- Authenticated web crawling and web scanner configuration tweaking
```
[the remark this answers is not in the document]
Comment (mjang-cobalt) [reply content/en/Getting started/Pentest objectives/Methodologies/web-methodologies.md:47]:
```suggestion
- Automated web crawling and web scanner configuration tweaking
```

**Pull request merged**